packages/tabs/stories/Tabs.stories.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Story, Meta } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { Tabs, TabList, Tab, TabPanel } from '@zendeskgarden/react-tabs';
+
+export default {
+  title: 'Components/Tabs',
+  subcomponents: { Tabs, TabList, Tab, TabPanel },
+  argTypes: {
+    orientation: {
+      control: {
+        type: 'inline-radio',
+        options: ['vertical', 'horizontal']
+      }
+    }
+  }
+} as Meta;
+
+interface IStoryProps {
+  orientation: 'vertical' | 'horizontal';
+  disabled: boolean;
+}
+
+export const Default: Story<IStoryProps> = ({ orientation, disabled }) => {
+  const [tab, setTab] = useState('elm');
+
+  return (
+    <Tabs
+      isVertical={orientation === 'vertical'}
+      selectedItem={tab}
+      onChange={newTab => {
+        action('test');
+        setTab(newTab);
+      }}
+    >
+      <TabList>
+        <Tab item="elm">Elm</Tab>
+        <Tab item="sugar-maple" disabled={disabled}>
+          Sugar Maple
+        </Tab>
+        <Tab item="dogwood" disabled={disabled}>
+          Sugar
+        </Tab>
```

Comment:
Follow existing example – https://zendeskgarden.github.io/react-components/tabs/#default-usage – by only disabling one tab. That gives us the ability to visually test keyboarding around a disabled tab.

Reply:
I based this off of the [example shown in the main Garden site](https://garden.zendesk.com/components/tabs#disabled). Do we not want these to match the states shown there?

Reply:
This is different. We do these storybook demos to push on boundaries vs. the website which provides examples to clearly demonstrate features/functionality.

Reply:
Removed.